frontend/components/site/SiteSettings/index.jsx

```diff
@@ -59,6 +59,11 @@ class SiteSettings extends React.Component {
 
     return (
       <div>
+        <p>
+          See our doucmentation site for more about
+          <a href="https://federalist-docs.18f.gov/pages/using-federalist/#managing-site-settings">these settings</a> or 
```

Comment:
Do we want these to open in another tab? Most of our external links on federalist currently do so. I think we should also provide a `title` attribute for these.

Reply:
Good point

Reply:
I should also have target="_blank" rel="noopener noreferrer" in the link right?

Reply:
yes! You'll probably have to put the properties on a separate line to stay within our linter's line length requirements

Reply:
yes